src/Graphics/PipelineCache.cs

```diff
@@ -26,6 +26,20 @@ namespace Microsoft.Xna.Framework.Graphics
 {
 	#region Internal PSO Hash Struct
 
+	internal sealed class StateHashComparer : IEqualityComparer<StateHash> {
```

Comment:
```suggestion
	internal sealed class StateHashComparer : IEqualityComparer<StateHash>
	{
```

Reply:
Thought I could commit this change via GitHub but for some reason it won't show the UI... this is the only thing left.

Reply:
Sorry for missing that one!